Taskfile.yml

```diff
@@ -63,9 +63,9 @@ tasks:
       - cmd: docker buildx prune --all
         ignore_error: true # build container may not exist
       - docker system prune --all --volumes
-      - bin/golangci-lint{{exeExt}} cache clean
```

Comment:
As far as I can see, `{{exeExt}}` is not needed, it looks to me that Windows can handle it without `.exe`